CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 [#460](https://github.com/plotly/dash-table/issues/460)
 - The `datestartswith` relational operator now supports number comparison
 - Fixed a bug where the implicit operator for columns was `equal` instead of the expected default for the column type
+[#546](https://github.com/plotly/dash-table/issues/546)
+- Visible columns are used correctly for both header and data rows
+
+### Added
+[#546](https://github.com/plotly/dash-table/issues/546)
+- New prop `export_columns` that takes values `all` or `visible` (default). This prop controls the columns used during export
```

Comment:
I'd prefer if `Added` came before `Fixed` in the changelog - new features >> 🐛 . (anyway as is it's splitting the `Fixed` section)